lib/app.go

```diff
@@ -78,12 +79,36 @@ func newApp(ra *schema.RuntimeApp, podManifest *schema.PodManifest, pod *pkgPod.
 		UserLabels:      ra.App.UserLabels,
 	}
 
+	var (
+		podVols       = podManifest.Volumes
+		podVolsByName = make(map[types.ACName]types.Volume, len(podVols))
+	)
```

Comment:
I would avoid this inline `var( ... )` block and just go for two plain `:=` assignments.